scripts/prepare_cardano_asset.py

```diff
@@ -0,0 +1,160 @@
+"""
+This script is a simple web scraper tool to prepare testing data for the `cardano/asset` endpoint.
```

Comment:
Add some notes how to run this script, what is needed to prepare (if something needed).
something like
```shell
SOME_ENV_VARS python3 prepare_cardano_asset.py